logstash-core/src/main/java/org/logstash/secret/store/backend/JavaKeyStore.java

```diff
@@ -38,101 +38,116 @@
  * <p>This class is threadsafe.</p>
  */
 public final class JavaKeyStore implements SecretStore {
```

Comment:
The diff is huge, but there shouldn't be anything but cosmetic changes here. (new methods, updates to the logging, other minor stuff) The core logic in use is the same as prior reviews.

Reply:
Thank you - this comment was helpful in reviewing this file! :)